From: Tomasz Buchert <tomasz@debian.org>
Date: Sun, 23 Apr 2017 19:40:32 +0200
Subject: Don't run readelf tests, because they are fragile and arch-specific.

---
 test/all_tests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/all_tests.py b/test/all_tests.py
index 4cb8e3c..6467af8 100755
--- a/test/all_tests.py
+++ b/test/all_tests.py
@@ -22,7 +22,7 @@ def main():
         return 1
     run_test_script('test/run_all_unittests.py')
     run_test_script('test/run_examples_test.py')
-    run_test_script('test/run_readelf_tests.py')
+    # run_test_script('test/run_readelf_tests.py')
 
 if __name__ == '__main__':
     sys.exit(main())
